src/main/groovy/com/github/erdi/gradle/webdriver/task/ConfigureBinary.groovy

```diff
@@ -68,7 +69,14 @@ abstract class ConfigureBinary extends DefaultTask {
         def versionAndUri = new DriverUrlsConfiguration(driverUrlsConfiguration.get().asFile()).versionAndUriFor(downloadSpec())
         def installer = new DriverDistributionInstaller(project, downloadRoot.asFile.get(), driverName, versionAndUri)
         def distributionRoot = installer.getDistributionRoot(versionAndUri.version).get()
-        def binaryFile = new File(distributionRoot, operatingSystem.getExecutableName(webDriverBinaryMetadata.binaryName))
+        def platformIndependentBinaryName = operatingSystem.getExecutableName(webDriverBinaryMetadata.binaryName)
+        def binaryFile = new File(distributionRoot, platformIndependentBinaryName)
+        if (!binaryFile.exists()) {
+            //Some versions of chromedriver hide the binary deeper, so we traverse the directory to find the binary
+            distributionRoot.traverse(type: FileType.FILES, nameFilter: platformIndependentBinaryName) { file ->
+                binaryFile = file
+            }
+        }
```

Comment:
Looks good in general but I would prefer to go with a native Gradle solution rather than to roll our own (untested but I'm fairly confident it will work):
```
        def binaryFile = project.objects.fileTree().tap {
            from(distributionRoot)
            include("**/$platformIndependentBinaryName")
        }.singleFile
```
This has a benefit of throwing if there is more than one file found and the logic is simpler because we don't special case nested files with `if (!binaryFile.exists()) {` while the performance penalty will be negligible.

Ideally, we would also not use `project.objects` as accessing the project object at execution time is incompatible with configuration cache and instead inject an instance of `ObjectFactory` via the constructor because [it's injectable](https://docs.gradle.org/current/userguide/custom_gradle_types.html#service_injection) into [custom Gradle types](https://docs.gradle.org/current/userguide/custom_gradle_types.html).